app/views/posts/_comment.html.slim

```diff
@@ -0,0 +1,20 @@
+.row.comment-partial
+  .col-sm-1
+    .thumbnail
+      - if comment.user.avatar.url.present?
```

Comment:
Try to research how to use `delegate` in rails, after that you can use `comment.user_avatar.url`